common/src/main/java/owmii/powah/api/PowahAPI.java

```diff
@@ -76,7 +78,41 @@ public static void registerHeatSource(ResourceLocation block, int heat) {
      * @return the heat of the block;
      **/
     public static int getHeatSource(Block block) {
-        return HEAT_SOURCES.getOrDefault(BuiltInRegistries.BLOCK.getKey(block), 0);
+        // Fix for Tinker's Construct compatibility or other mods that use LiquidBlock
```

Comment:
You can just add the liquid block directly. See discussion https://github.com/Technici4n/Powah/pull/229.